generated/nidcpower/nidcpower/session.py

```diff
@@ -6653,6 +6653,8 @@ def set_sequence(self, values, source_delays):
                 The valid values are between 0 and 167 seconds.
 
         '''
+        if source_delays is not None and len(source_delays) != len(values):  # case S160
```

Comment:
It's worth noting that if you pass `values: None, source_delays: [1.0]` as the arguments then this will raise `TypeError` when calling `len(None)`. This is equally true of the existing released code, so this is not a regression. It's also not that important IMO since it should result in an exception regardless. Just something to consider.